v1/runGame.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Max
+ * Date: 05.02.2016
+ * Time: 10:26
+ */
+
+
+require_once "lib/gamefieldcontroller.php";
+require_once "lib/gamefield.php";
+require_once "lib/consoleOutput.php";
+
+require '../vendor/autoload.php';
+
+
+
+use Ulrichsg\Getopt;
+$options = new Getopt\Getoptv2(array(
+            new Getopt\Option('o','output',Getopt\Getoptv2::REQUIRED_ARGUMENT),
+            new Getopt\Option('w','width',Getopt\Getoptv2::REQUIRED_ARGUMENT),
+            new Getopt\Option('h','height',Getopt\Getoptv2::REQUIRED_ARGUMENT),
+            new Getopt\Option('c','numCycles',Getopt\Getoptv2::REQUIRED_ARGUMENT),
+            new Getopt\Option('t','type',Getopt\Getoptv2::REQUIRED_ARGUMENT)
+));
+
+$options->parse();
+
+
+if ($options->getOption('w')) $x = $options->getOption('h');
+else die("Argument --width is missing");
```

Comment:
I do not think that this is needed because you are saying above, that this options are required...Please check what getopt is doing, if you do not add this options!